src/index.ts

```diff
@@ -62,7 +62,8 @@ export async function run({ git, forge, config }: { git: SimpleGit; forge: Forge
   await git.branch(['--set-upstream-to', `origin/${releaseBranch}`]);
   await git.pull();
 
-  const isReleaseCommit = config.ci.commitMessage?.startsWith(config.ci.releasePrefix);
+  const isReleaseCommit = config.ci.commitMessage?.startsWith(config.ci.releasePrefix) ||
+    config.ci.commitMessage?.startsWith(`Merge pull request '${config.ci.releasePrefix}`);
```

Comment:
Could you provide some examples how those commit messages would look like. Might be better to use sth like `.replace(/Merge pull request/i, '').startsWith(`

Github for example has:

```
Merge pull request #3725 from woodpecker-ci/plugin-ready-release-go
```

Reply:
For Gitea/Forgjo, the example is `Merge pull request '🎉 Release 3.8.5' (#13) from next-release/main into main`, which is why I added the `releasePrefix` too so it selects those. Note the single quotes for the Release message too.